docs/cluster-configuration.md

```diff
@@ -343,3 +344,98 @@ policies:
+        * `metadata.name` is updated from `sno-ran-du-pg-v4-Y-Z-v2` to `sno-ran-du-pg-v4-Y-Z-v3` (the `ztp-sno-ran-du-v4-Y-Z` namespace is kept).
+        * `policyDefaults.placement.labelSelector.sno-ran-du-policy` is updated from `v2` to `v3` such that the policy binding is updated.
+        * All policy names are updated from `v2` to `v3` (example: `v2-subscriptions-policy` -> `v3-subscriptions-policy`).
+        * The following manifests are added under the `v3-subscriptions-policy`:
+        ```yaml
+        - path: source-crs/SriovNetworkNodePolicy-SetSelector.yaml
+          patches:
+          - metadata:
+              name: "sriov-nnp-du-fh"
+            spec:
+              deviceType: netdevice
+              isRdma: false
+              nicSelector:
+                  pfNames: '{{hub fromConfigMap "" (printf "%s-pg" .ManagedClusterName) "sriov-network-pfNames-1" | toLiteral hub}}'
+              nodeSelector:
+                  node-role.kubernetes.io/master: ""
+              numVfs: 8
+              priority: 10
+              resourceName: du_fh
+
+        - path: source-crs/SriovNetworkNodePolicy-SetSelector.yaml
+          patches:
+          - metadata:
+              name: "sriov-nnp-du-mh"
+            spec:
+              deviceType: vfio-pci
+              isRdma: false
+              nicSelector:
+                  pfNames: '{{hub fromConfigMap "" (printf "%s-pg" .ManagedClusterName) "sriov-network-pfNames-2" | toLiteral hub}}'
+              nodeSelector:
+                  node-role.kubernetes.io/master: ""
+              numVfs: 8
+              priority: 10
+              resourceName: du_mh
+        ```
+
+    * A new version of the [policytemplate-defaults-v1](samples/git-setup/clustertemplates/version_4.Y.Z/sno-ran-du/policytemplates-defaults-v1.yaml) ConfigMap is created - [policytemplate-defaults-v2](samples/git-setup/clustertemplates/version_4.Y.Z/sno-ran-du/policytemplates-defaults-v2.yaml):
+        * `metadata.name` is updated from `policytemplate-defaults-v1` to `policytemplate-defaults-v2`.
+        * update the defaults to reflect the new schema and thus the needed configuration values, in our case: `sriov-network-pfNames-1` and `sriov-network-pfNames-2`.
+    
+    * Create a new version of the [clusterinstance-defaults-v3](samples/git-setup/clustertemplates/version_4.Y.Z/sno-ran-du/clusterinstance-defaults-v3.yaml) `ConfigMap` - [clusterinstance-defaults-v4](samples/git-setup/clustertemplates/version_4.Y.Z/sno-ran-du/clusterinstance-defaults-v4.yaml):
+        * Update the name to `clusterinstance-defaults-v4` (the namespace stays `sno-ran-du-v4-Y-Z`).
+        * Update the `sno-ran-du-policy` ManagedCluster `extraLabel` from `v2` to `v3`.
+
+        **Note:** An alternative to creating a new ConfigMap, is to change the label directly in the ProvisioningRequest by updating `spec.templateParameters.clusterInstanceParameters.extraLabels.ManagedCluster.sno-ran-du-policy` from `v2` to `v3`.
```

Comment:
I don't think we should mention this as an alternative - the SMO should only have to switch to the new cluster template version.